plugins/woocommerce-blocks/assets/js/blocks/product-collection/edit/inspector-controls/index.tsx

```diff
@@ -91,8 +92,9 @@ const ProductCollectionInspectorControls = (
 		isArchiveTemplate && shouldShowFilter( CoreFilterNames.INHERIT );
 	const showFilterableControl =
 		! isArchiveTemplate && shouldShowFilter( CoreFilterNames.FILTERABLE );
-	const showOrderControl =
+	const showCustomOrderControl =
 		showQueryControls && shouldShowFilter( CoreFilterNames.ORDER );
+	const showDefaultOrderControl = inherit === true;
```

Comment:
Nitpick: What about renaming `showQueryControls` to `showCustomQueryControls`. And in that case I'd make the next condition as such:

```suggestion
	const showCustomOrderControl =
		showCustomQueryControls && shouldShowFilter( CoreFilterNames.ORDER );
	const showDefaultOrderControl = ! showCustomQueryControls;
```

This is to show explicitly they're mutually exclusive so we either show one or another.

Reply:
Good point, done in a11d09e7655179c1021917bb22d619f7262f38bc.